src/external/clang.BUILD

```diff
@@ -63,3 +63,8 @@ filegroup(
         "lib/libunwind.a",
     ],
 )
+
+sh_binary(
```

Comment:
Why is this needed? So we can run formatting as a bazel target?

Reply:
yep. We could go back to just committing the binary, but I think this helps in terms of being able to trace where files are coming from.